README.md

```diff
@@ -147,49 +147,24 @@ To use the GitHub Action with Codacy integration:
               max-allowed-issues: 2147483647
     ```
 
-4.  Optionally, [enable standalone client side tools](https://docs.codacy.com/related-tools/local-analysis/client-side-tools/) by passing the following options to the action.
+4.  Optionally, specify the following parameters to run [**standalone** client-side tools](https://docs.codacy.com/related-tools/local-analysis/client-side-tools/):
 
     ```yaml
-    name: Codacy Analysis CLI
-
-    on: ["push"]
-
-    jobs:
-      codacy-analysis-cli:
-        name: Codacy Analysis CLI
-        runs-on: ubuntu-latest
-        steps:
-          # ...
-          - name: Run Codacy Analysis CLI
-            uses: codacy/codacy-analysis-cli-action@master
-            with:
-              # ...
-              run-gosec: "true"
-              run-staticcheck: "true"
+    run-gosec: "true"
+    run-staticcheck: "true"
     ```
 
-    In case you just want to run the standalone client side tools you can disable the dockerized tools by passing the `run-docker-tools: "false"` to the action.
-
-    In the case of Clang Tidy and Faux Pas, due to the complex orchestration of the tool, this action will instead receive the path to a file with the outputs of the tool and will then push them to Codacy.
-    To pass the file use the configurations bellow:
+    Due to the complex orchestration of the tools Clang-Tidy and Faux Pas, in this case the action can receive instead the output files of the tools and uploads them to Codacy:
 
     ```yaml
-    name: Codacy Analysis CLI
+    clang-tidy-output: "path/to/output"
+    faux-pas-output: "path/to/output"
+    ```
 
-    on: ["push"]
+    If you only want to run the standalone client-side tools and not all the containerized tools supported by Codacy, specify:
```

Comment:
Here I try to clarify that unless this option exists, the GitHub Action will run **all tools supported by Codacy** and not just the client-side ones [listed on this page](https://docs.codacy.com/related-tools/local-analysis/client-side-tools/) - do you confirm that this is correct?